CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# Contributing to ProtoSchool
+
+### What is ProtoSchool?
+
+ProtoSchool is an educational community dedicated to making it easy to get started with decentralized technologies. It's an initiative of [Protocol Labs](https://protocol.ai/), which sponsors such open source projects as [IPFS](https://ipfs.io/), [IPLD](https://ipld.io/), [libp2p](https://libp2p.io/), and [Multiformats](https://multiformats.io/). However, we depend on volunteers to develop our coding workshops and make our content available at chapters around the world through live events. We always need more volunteers, and there are always things to do!
```

Comment:
A few things about the language here. First, we shouldn't use the term "volunteer" as it implies a relationship in which they give and we take :)

We also don't need to mention Protocol Labs. We don't mention in PL in our other projects, we give them their own project identities and are open to leaders coming in from outside of PL.

I like that we're actively inviting/soliciting contributions here though. Let's just move towards language that describes ProtoSchool as a community driven effort in which anyone who would like to participate can be come an active member and event decision maker in the project.

Reply:
Without mentioning Protocol Lab, do you think I should still list out those projects we're interested in covering to avoid scope creep into Dat, Beaker, etc ?

Reply:
I mean, we're open to workshops from other projects, we just won't be doing the work to write them ourselves. We may want to define some clear scope boundaries somewhere else, but for the purposes of relatively new people I think we can leave it open.